openslides/topics/models.py

```diff
@@ -17,19 +18,26 @@ class Topic(RESTModelMixin, models.Model):
     text = models.TextField(blank=True)
     attachments = models.ManyToManyField(Mediafile, blank=True)
 
+    # In theory there could be one then more agenda_item. But support only one.
+    # See the property agenda_item.
+    agenda_items = GenericRelation(Item)
+
     class Meta:
         default_permissions = ()
 
     def __str__(self):
         return self.title
 
+    @classmethod
+    def all_query(cls):
+        return cls.objects.prefetch_related('attachments', 'agenda_items')
+
```

Comment:
I want to add it to every model which Serializer needs more then one model. For example if the user would add the name of every group in the get_full_data() method, then the user model would also need this method.

I have not tested yet which models need the method at the moment. Maybe it is only the agenda related models, maybe not.